osrdyne/src/drivers/kubernetes/keda.rs

```diff
@@ -44,6 +44,6 @@ pub enum MetricType {
 pub struct Trigger {
     pub type_: String,
     pub metadata: HashMap<String, String>,
-    pub use_cached_metrics: bool,
+    pub use_cached_metrics: Option<bool>,
```

Comment:
Why do we need that to be optional now? Can't understand why, and especially, I don't see the link with the current PR.

Reply:
It is related to this PR because we deserialize objects that does not have the field and we fail badly (meaning : osrdyne crash).